File: Domain/AppUser.cs
using Microsoft.AspNetCore.Identity;
using System.ComponentModel.DataAnnotations;
namespace Domain
{
    public class AppUser : IdentityUser
    {
        [StringLength(100)]
        public string Name { get; set; }
        [StringLength(100)]
        public string Surname { get; set; }
        public bool IsAdmin {get;set;} //letting the API send request to determine if the user is admin
    }
}

File: client-app/src/Models/appUser.ts
export interface AppUser{
    id:string; 
    name:string;
    surname:string;
    email:string;
    isAdmin:string;
    password:string;
}

export  class AppFormValues {
    id?:string=''; 
    name:string='';
    surname:string='';
    email:string='';
    isAdmin:string='false';
    password?:string='';
    constructor(values?:AppFormValues) {

       if(values)
       {
        this.id = values.id;
        this.email = values.email;
        this.name = values.name;
        this.surname = values.surname;
        this.isAdmin = values.isAdmin;
        this.password =values.password;
       }
    }

}